### PR TITLE
[zumly] Add types for zumly

### PR DESCRIPTION
**Pull request by edwinork**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [x] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/zumly/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
+// Type definitions for zumly 0.9
+// Project: https://zumly.org
+// Definitions by: edwinork <https://github.com/edwinork>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare global {
+    /**
+     * Zumly powers your apps with a zoomable user interface (ZUI) taste.
+     */
+    class Zumly {
```
Comment (rbuckton) [inline types/zumly/index.d.ts:10]: When used as a script in the browser, the global name is `zumly` not `Zumly`: https://unpkg.com/browse/zumly@0.9.9/dist/zumly.umd.js (line 9)

**Review by rbuckton — CHANGES_REQUESTED** [review]

File: types/zumly/index.d.ts (hunk 1 above)
Comment (rbuckton) [reply types/zumly/index.d.ts:10]: I verified this by modifying the [codepen linked on the site](https://codepen.io/zumly/pen/gOPQovd) to use `import "https://unpkg.com/zumly@0.9.9/dist/zumly.umd.js";` and had to modify the call to `new Zumly` to use `new zumly` for the codepen to work when loaded as a script.

Comment (edwinork) [reply types/zumly/index.d.ts]: Thanks for the catch! Fixed.

File: types/zumly/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                // Transition ease. Default 'ease-in-out'
+                ease: string;
+            };
+            // Activate debug notifications
+            debug?: boolean;
+        }
+    }
+}
+
+export default zumly;
```
Comment (andrewbranch) [inline types/zumly/index.d.ts:63]: This is a UMD module, meaning it’s available as a global, through CommonJS, or through AMD. You covered the global case manually but the `export default` is wrong. What you want to do is delete the `declare global`, rename the namespace to lowercase `zumly` like the class, replace the `export default` with `export = zumly`, and add `export as namespace zumly` at the end.

**Review by andrewbranch — CHANGES_REQUESTED** [review]

File: types/zumly/index.d.ts (hunk 2 above)
Comment (edwinork) [reply types/zumly/index.d.ts]: I appreciate your help! I think I got confused with trying to make it available as both ES6 module and UMD to match [this example](https://github.com/zumly/zumly#setup).

I pushed the changes requested by you. Please take a look when you have time.

**Review by andrewbranch — APPROVED** [review]
Looks perfect! Module structure is the trickiest thing about DefinitelyTyped. Congrats on your first contribution here!

**Comment by edwinork** [comment]
Ready to merge

**Pull request merged**